products/motorola-mobility.md

```diff
@@ -19,6 +19,19 @@ eolColumn: Security Updates
 #
 
 releases:
+- releaseCycle: "moto-e15"
+  releaseLabel: "Motorola Moto E15"
+  releaseDate: 2024-12-01
```

Comment:
Where can I confirm this date ? On https://www.gsmarena.com/motorola_moto_e15-13574.php I see 2025-01-22.